modin/pandas/groupby.py

```diff
@@ -124,7 +124,7 @@ def skew(self, *args, **kwargs):
             type(self._query_compiler).groupby_skew,
             agg_args=args,
             agg_kwargs=kwargs,
-            numeric_only=True,
+            numeric_only=NumericOnly.TRUE_EXCL_NUMERIC_CATEGORIES,
```

Comment:
previously, native pandas `.skew()` implementation called by `qc.groupby_agg` was actually dropping unsuitable categorical columns, now we need to drop them manually